### PR TITLE
Removal of SBO content from various topic maps

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

enterprise-4.16 ++

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-6051

Link to docs preview:

Removal of the Service Binding Operator row from the Feature Summary table:

https://76287--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/oke_about.html#feature-summary

Removal of the Connecting applications to services content section from the ROSA distribution: https://76287--ocpdocs-pr.netlify.app/openshift-rosa/latest/applications/creating_applications/odc-creating-applications-using-developer-perspective

Removal of the Connecting applications to services content section from the Openshift dedicated distribution: https://76287--ocpdocs-pr.netlify.app/openshift-dedicated/latest/applications/odc-viewing-application-composition-using-topology-view

Removal of the Connecting applications to services content section from the Openshift enterprise distribution: https://76287--ocpdocs-pr.netlify.app/openshift-enterprise/latest/applications/odc-exporting-applications

ACK from Eng: @sadlerap @dperaza4dustbit 
ACK from QE: @pmacik 
ACK from CS: @Preeticp/@Srivaralakshmi 
ACK from PM: @harrietgrace 
ACK from internal peer reviewer: @eromanova97 
ACK from peer reviewer: @Srivaralakshmi 

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by sadlerap — COMMENTED** [review]
one question, otherwise lgtm

File: applications/index.adoc
Comment (Dhruv-Soni11) [reply applications/index.adoc]: Now that I see this again, leaving this trailing information in the document does not make sense. I will remove the entire section from the content as it will be the best course of action for now.

**Comment by Srivaralakshmi** [comment]
@Dhruv-Soni11 Good work and great collaboration! Please add the QE and internal peer reviewers here for review once you have an LGTM/approval from @sadlerap. Thanks!

**Comment by Dhruv-Soni11** [comment]
> @Dhruv-Soni11 Good work and great collaboration! Please add the QE and internal peer reviewers here for review once you have an LGTM/approval from @sadlerap. Thanks!

Hi @Srivaralakshmi - I have added the internal peer reviewer information and I'm just waiting for confirmation for the QE person from @sadlerap and the others.

**Review by sadlerap — APPROVED** [review]
lgtm

**Review by dperaza4dustbit — APPROVED** [review]

**Review by Srivaralakshmi — COMMENTED** [review]
@Dhruv-Soni11 LGTM

**Comment by balajisiva** [comment]
I am ok with this as the decision to deprecate this was made before I even joined Red Hat due to low adoption.  There has been zero interest that I am aware of.

**Review by dperaza4dustbit — COMMENTED** [review]
lgtm

**Pull request merged**
